scripts/build.bat

```diff
@@ -42,13 +42,13 @@ cd ..
 rem package config-service and admin-service
 echo "==== starting to build config-service and admin-service ===="
 
-call mvn clean package -DskipTests -pl apollo-configservice,apollo-adminservice -am -Dapollo_profile=github -Dspring_datasource_url=%apollo_config_db_url% -Dspring_datasource_username=%apollo_config_db_username% -Dspring_datasource_password=%apollo_config_db_password%
+call mvn clean package -DskipTests -pl apollo-configservice,apollo-adminservice -am -Dapollo_profile=github -Dspring_datasource_url=%apollo_config_db_url% -Dspring_datasource_username=%apollo_config_db_username% -Dspring_datasource_password=%apollo_config_db_password% -Dspring.profiles.group.github=%spring_profiles_group_github%
```

Comment:
It looks like the `spring_profiles_group_github` is not assigned any value?

Reply:
@nobodyiam Fixed